gatekeeper/metadata.csv

```diff
@@ -8,12 +8,15 @@ gatekeeper.constraint_template_ingestion.count,count,,,,Total number of constrai
 gatekeeper.violations,gauge,,,,Total number of violations per constraint,0,gatekeeper,gk.violations,
 gatekeeper.constraints,gauge,,,,Current number of known constraints,0,gatekeeper,gk.constraints,
 gatekeeper.constraint_templates,gauge,,,,Number of observed constraint templates,0,gatekeeper,gk.constraint_tmpl,
-gatekeeper.request.duration.seconds.sum,count,,second,,The response time in seconds,1,gatekeeper,gk.duration.sum,
-gatekeeper.request.duration.seconds.count,count,,second,,The response time in seconds,1,gatekeeper,gk.duration.cnt,
-gatekeeper.request.count,count,,,,Total number of requests that are routed to webhook,1,gatekeeper,gk.req_cnt,
+gatekeeper.request.duration.seconds.sum,count,,second,,[Deprecated since Gatekeeper v3.4.0] The response time in seconds,1,gatekeeper,gk.duration.sum,
+gatekeeper.request.duration.seconds.count,count,,second,,[Deprecated since Gatekeeper v3.4.0] The response time in seconds,1,gatekeeper,gk.duration.cnt,
+gatekeeper.request.count,count,,,,[Deprecated since Gatekeeper v3.4.0] Total number of requests that are routed to webhook,1,gatekeeper,gk.req_cnt,
 gatekeeper.sync,gauge,,,,Total number of resources of each kind being cached,0,gatekeeper,gk.sync,
 gatekeeper.sync.duration.seconds.sum,count,,second,,Latency of sync operation in seconds,1,gatekeeper,gk.sync_duration.sum,
 gatekeeper.sync.duration.seconds.count,count,,second,,Latency of sync operation in seconds,1,gatekeeper,gk.sync_duration.cnt,
 gatekeeper.sync.last_run_time,gauge,,,,Timestamp of last sync operation,0,gatekeeper,gk.sync_last,
 gatekeeper.watch.intended,gauge,,,,The total number of Group/Version/Kinds that the watch manager has instructions to watch,0,gatekeeper,gk.watch_intented,
 gatekeeper.watch.watched,gauge,,,,The total number of Group/Version/Kinds currently watched by the watch manager,0,gatekeeper,gk.watch_watched,
+gatekeeper.validation.request.count,count,,,,The number of requests that are routed to validation webhook,1,gatekeeper,gk.val_req_cnt,
+gatekeeper.validation.request.duration.seconds.sum,count,,second,,The response time in second,1,gatekeeper,gk.val_duration.sum,
+gatekeeper.validation.request.duration.seconds.count,count,,second,,The response time in second,1,gatekeeper,gk.val_duration.cnt,
```

Comment:
Should we also get the `_bucket` metric?

Reply:
In OpenMetrics v1, the buckets are ingested as the `upper_bound` tag in the `.count` metrics